bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/util/ModifyTableBuilder.java

```diff
@@ -48,70 +39,34 @@ public static ModifyTableBuilder create() {
   }
 
   /**
-   * This method will build list of of protobuf {@link ModifyColumnFamiliesRequest.Modification} objects based on a diff of the
+   * This method will build {@link ModifyColumnFamiliesRequest} objects based on a diff of the
    * new and existing set of column descriptors.  This is for use in
    * {@link org.apache.hadoop.hbase.client.Admin#modifyTable(TableName, HTableDescriptor)}.
    */
-  public static ModifyTableBuilder buildModifications(
-      HTableDescriptor newTableDescriptor,
-      HTableDescriptor currentTableDescriptors) {
-    ModifyTableBuilder builder = new ModifyTableBuilder();
+  public static ModifyColumnFamiliesRequest buildModifications(
+          HTableDescriptor newTableDescriptor,
+          HTableDescriptor currentTableDescriptors,
+          ModifyColumnFamiliesRequest request) {
     Set<String> currentColumnNames = getColumnNames(currentTableDescriptors);
     Set<String> newColumnNames = getColumnNames(newTableDescriptor);
 
     for (HColumnDescriptor hColumnDescriptor : newTableDescriptor.getFamilies()) {
-      if (currentColumnNames.contains(hColumnDescriptor.getNameAsString())) {
-        builder.modify(hColumnDescriptor);
+      String columnName = hColumnDescriptor.getNameAsString();
+      if (currentColumnNames.contains(columnName)) {
+        request.updateFamily(columnName,
+                buildGarbageCollectionRule(hColumnDescriptor));
```

Comment:
Please use a single variable for the result of `buildGarbageCollectionRule(hColumnDescriptor)` before the `if`.